.ko.yaml

```diff
@@ -0,0 +1,5 @@
+---
+defaultLdflags:
+  - -X main.version={{ .Git.Tag }}
+  - -X main.commit={{ .Git.FullCommit }}
+  - -X main.date={{ .Date }}
```

Comment:
I like how this is done really much.
I will point to the docs in case someone else is interested on this part: https://ko.build/configuration/

Reply:
Sorry I didn't wait for your review, I missed you added yourself.